cookbooks/bcpc-hadoop/templates/default/hb_hbase-site.xml.erb

```diff
@@ -166,4 +166,9 @@
     <value>org.apache.hadoop.hbase.ipc.SecureRpcEngine</value>    
   </property>   
   <% end %>
+
+  <property>
+    <name>hbase.replication</name>
```

Comment:
The default value for ``base.replication`` in ``hbase`` is ``true``. This is to make it explicit in ``hbase-site`` so that users are aware and doesn't change the default behavior of the cluster.